templates/theme-default/package.json

```diff
@@ -33,7 +33,7 @@
   "devDependencies": {
     "@babel/core": "^7.16.0",
     "@jungvonmatt/contentful-fakes": "^1.7.1",
-    "@jungvonmatt/contentful-migrations": "^5.3.0",
+    "@jungvonmatt/contentful-migrations": "^5.3.1",
     "@jungvonmatt/contentful-ssg": "^1.7.1",
```

Comment:
We should also bump `@jungvonmatt/contentful-ssg` to `^1.9.0`

Reply:
I have updated them to the latest version for now. I think we can clean up both package.json files in app and theme in general. I think there are other scripts and packages that are obsolete.